premake5.lua

```diff
@@ -88,6 +90,11 @@ filter({"configurations:Release", "platforms:Windows"})
   linkoptions({
     "/NODEFAULTLIB:MSVCRTD",
   })
+  buildoptions({
```

Comment:
It'd be nice if you could explain what these build options actually do in a comment so we don't have to reference MSDN docs for it.

Reply:
Ahh, yes. They are because premake doesn't have an option for those. I'll add in a comment.